src/PrestaShopBundle/Resources/config/routing/admin/improve/design/cms_pages.yml

```diff
@@ -0,0 +1,77 @@
+admin_cms_pages_index:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:index'
+        _legacy_controller: AdminCmsContent
+
+admin_cms_pages_search_cms_category:
+    path: /category/search
+    methods: POST
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:searchCategory'
+        _legacy_controller: AdminCmsContent
+
+admin_cms_pages_create_cms_category:
+    path: /category/create
+    methods: [GET, POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:createCmsCategory'
+        _legacy_controller: AdminCmsContent
+
+admin_cms_pages_edit_cms_category:
+    path: /category/{cmsCategoryId}/edit
+    methods: [GET, POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Improve\Design\CmsPage:editCmsCategory'
+        _legacy_controller: AdminCmsContent
+    requirements:
+        cmsCategoryId: \d+
+
+admin_cms_pages_delete_cms_category:
+    path: /category/{cmsCategoryId}/delete
+    methods: DELETE
```

Comment:
i prefer to be consistent and use `[DELETE]` even though it accepts only 1 method, but its personal preference so i cant force that. :)

p.s. also notice method**s**, its in plural, so maybe array makes sense. :p